turbine/src/broadcast_stage/standard_broadcast_run.rs

```diff
@@ -510,7 +510,9 @@ impl BroadcastRun for StandardBroadcastRun {
 fn should_use_merkle_variant(slot: Slot, cluster_type: ClusterType, shred_version: u16) -> bool {
     match cluster_type {
         ClusterType::Testnet => shred_version == 28353,
-        _ => (slot % 19) == 1,
+        ClusterType::MainnetBeta => (slot % 19) == 1,
+        ClusterType::Devnet => true,
```

Comment:
Is there a reason why we should be gating merkle shreds by shred version on `Testnet` but not `Devnet`? Should the shred version check for testnet be removed?

Reply:
Ha, we both asked this at the same time

Reply:
It was an extra sanity check that we are on the right cluster, but I don't think we need it anymore.
Removed.